.travis.yml

```diff
@@ -1,15 +1,20 @@
+###
```

Comment:
This comment is redundant in my option. I'd suggest removing it.

Reply:
Not redundant on a printout.